woocommerce/woocommerce-gutenberg-products-block
# Category List block: add support for global style

~~This PR is blocked by #5515~~

This PR adds support for global style for the component Product Title.

Now, the user can edit the style for:

- font-size
- link color
- text color


Fixes #4965 




### Screenshots


![image](https://user-images.githubusercontent.com/4463174/148087612-2dc762a9-5d12-46f0-aa81-c9dd4b975dcf.png)
*without global style*

![image](https://user-images.githubusercontent.com/4463174/148087492-49038b4d-b6fb-48db-9fd2-2d65a36d64fc.png)
*with global style*

### Testing

### Manual Testing

Check out this branch:

1. On `WordPress 5.9`, install and enable the `Gutenberg` plugin.
2. Install and enable the `Twenty Twenty-Two` theme.
3. Add the `Single Product block` (this block contains `Product Title Block`) to a post.
4. Get the focus on `Category List Block`.
5. On the right sidebar, personalize the styles of the block.
6. Go on the page and check if there are changes.
7. Reset to default using the `Reset` button from the different sections. 
8. Go to Dashboard and select Appearance > Editor (beta). On top of the screen, select Home > Browser all templates > Single Post. When the page is loaded, add the block to the page.
9. On the Editor page click on the `Styles` icon on the right-top corner.
10. Verify that the `Category List Block` is shown under the `Blocks` section. Personalize again the block.
11. Save your changes.
12. Go on the page created earlier and check if all styles are applied correctly.
13. Edit your previous post/page again.
14. Change again the styles.
15. Save your changes.
16. Check if these styles have priority over the styles from the Site Editor.



### Changelog

> Category List block: add support for global style.
